vm_supervisor/conf.py

```diff
@@ -70,57 +70,57 @@ class Settings(BaseSettings):
     START_ID_INDEX: int = 4
     PREALLOC_VM_COUNT: int = 0
     REUSE_TIMEOUT: float = 60 * 60.0
-    WATCH_FOR_MESSAGES: bool = True
-    WATCH_FOR_UPDATES: bool = True
-    NETWORK_INTERFACE: str = "eth0"
+    WATCH_FOR_MESSAGES = True
+    WATCH_FOR_UPDATES = True
+    NETWORK_INTERFACE = "eth0"
     DNS_RESOLUTION: Optional[DnsResolver] = DnsResolver.resolv_conf
     DNS_NAMESERVERS: Optional[List[str]] = None
 
-    API_SERVER: str = "https://api2.aleph.im"
-    USE_JAILER: bool = True
+    API_SERVER = "https://api2.aleph.im"
+    USE_JAILER = True
     # System logs make boot ~2x slower
-    PRINT_SYSTEM_LOGS: bool = False
+    PRINT_SYSTEM_LOGS = False
     # Networking does not work inside Docker/Podman
-    ALLOW_VM_NETWORKING: bool = True
-    FIRECRACKER_PATH: str = "/opt/firecracker/firecracker"
-    JAILER_PATH: str = "/opt/firecracker/jailer"
-    LINUX_PATH: str = "/opt/firecracker/vmlinux.bin"
-    INIT_TIMEOUT: float = 20
+    ALLOW_VM_NETWORKING = True
+    FIRECRACKER_PATH = "/opt/firecracker/firecracker"
+    JAILER_PATH = "/opt/firecracker/jailer"
+    LINUX_PATH = "/opt/firecracker/vmlinux.bin"
+    INIT_TIMEOUT: float = 20.
```

Comment:
While this works, wouldn't 20.0 be clearer?

Reply:
I have the habit to write floats like this, especially when the decimals are not relevant.